src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -4,7 +4,10 @@
       <!-- Row Title -->
       <v-col cols="3">
         <label :class="{'error-text': invalidSection}"><strong>Business Type</strong></label>
-        <v-col md="1">
+        <v-col
+          md="1"
+          class="pa-0"
+        >
```

Comment:
Check this on a narrow viewport (screen). Generally, columns should stack without weird whitespace.

(Do you have screenshots of normal vs narrow?)

Reply:
this is what it looks like before:
<img width="996" alt="Pasted Graphic" src="https://github.com/bcgov/business-edit-ui/assets/16507223/77f7a071-0acb-433f-8b68-d7c6ce9e5774">
after I added class="pa-0”: (normal)
<img width="996" alt="Pasted Graphic 1" src="https://github.com/bcgov/business-edit-ui/assets/16507223/451276df-0470-4e22-893e-c1d8bbebd657">
(narrow):
<img width="397" alt="Pasted Graphic 2" src="https://github.com/bcgov/business-edit-ui/assets/16507223/f458418a-54ab-4f7d-bdb1-b01ec24ce323">

Reply:
That's good. Thanks.